eng/TestRunsheetBuilder/TestRunsheetBuilder.targets

```diff
@@ -38,6 +38,9 @@
       <_CreateRunsheet Condition=" '$(_RequiresPackages)' != 'true' and '$(FullE2e)' != 'true' ">true</_CreateRunsheet>
 
       <_CreateRunsheet Condition=" '$(MSBuildProjectName)' == 'Aspire.Templates.Tests' ">false</_CreateRunsheet>
+
+      <!-- FIXME: temporary workaround for https://github.com/Azure/azure-functions-dotnet-worker/issues/2969 -->
+      <_PreCommand Condition=" '$(MSBuildProjectName)' == 'Aspire.Playground.Tests' ">./dotnet.sh build &quot;%24(pwd)/playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.Functions/AzureFunctionsEndToEnd.Functions.csproj&quot; -c $(Configuration) /p:SkipUnstableEmulators=true &amp;&amp; </_PreCommand>
```

Comment:
Now that we have these command lines being generated per project, we should have this kind of project-specific stuff in the test projects themselves.

Reply:
This crossed my mind too, but I'm unsure how discoverable that may be.

Reply:
This is not going to be a common thing to do. And documentation can help with discoverability. But the customizations belong with the specific projects, IMHO.